canonical-web-and-design/ubuntu.com
# Account users/Add organisation title

## Done

- Add organisation title
- add @testing-library/react
- add AccountUsers test
- remove cypress workaround
- update tsconfig.json

## QA
- log in
- go to /advantage/users
- verify "myOrganisation" is displayed

## Screenshots

![image](https://user-images.githubusercontent.com/7452681/131979386-928bb66a-a91d-4bda-aa12-ef53a81cc9f7.png)

## Design reference
https://zpl.io/aRw0wRK
